<ui version="4.0" >
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>JACK Audio Connection Kit - Qt GUI Interface.

   Copyright (C) 2003-2014, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

 </comment>
 <class>qjackctlConnectionsForm</class>
 <widget class="QWidget" name="qjackctlConnectionsForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>1</hsizetype>
    <vsizetype>1</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font" >
   <font/>
  </property>
  <property name="windowTitle" >
   <string>Connections - JACK Audio Connection Kit</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="qjackctl.qrc" >:/images/connections1.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>4</number>
   </property>
   <property name="spacing" >
    <number>4</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="ConnectionsTabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="AudioConnectTab" >
      <attribute name="title" >
       <string>Audio</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>4</number>
       </property>
       <property name="spacing" >
        <number>4</number>
       </property>
       <item row="0" column="0" >
        <widget class="qjackctlConnectView" native="1" name="AudioConnectView" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::TabFocus</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>4</number>
         </property>
         <property name="spacing" >
          <number>4</number>
         </property>
         <item>
          <widget class="QPushButton" name="AudioConnectPushButton" >
           <property name="toolTip" >
            <string>Connect currently selected ports</string>
           </property>
           <property name="text" >
            <string>&amp;Connect</string>
           </property>
           <property name="icon" >
            <iconset resource="qjackctl.qrc" >:/images/connect1.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="AudioDisconnectPushButton" >
           <property name="toolTip" >
            <string>Disconnect currently selected ports</string>
           </property>
           <property name="text" >
            <string>&amp;Disconnect</string>
           </property>
           <property name="icon" >
            <iconset resource="qjackctl.qrc" >:/images/disconnect1.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="AudioDisconnectAllPushButton" >
           <property name="toolTip" >
            <string>Disconnect all currently connected ports</string>
           </property>
           <property name="text" >
            <string>Disconnect &amp;All</string>
           </property>
           <property name="icon" >
            <iconset resource="qjackctl.qrc" >:/images/disconnectall1.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="AudioExpandAllPushButton" >
           <property name="toolTip" >
            <string>Expand all client ports</string>
           </property>
           <property name="text" >
            <string>E&amp;xpand All</string>
           </property>
           <property name="icon" >
            <iconset resource="qjackctl.qrc" >:/images/expandall1.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="AudioRefreshPushButton" >
           <property name="toolTip" >
            <string>Refresh current connections view</string>
           </property>
           <property name="text" >
            <string>&amp;Refresh</string>
           </property>
           <property name="icon" >
            <iconset resource="qjackctl.qrc" >:/images/refresh1.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="MidiConnectTab" >
      <attribute name="title" >
       <string>MIDI-JACK</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>4</number>
       </property>
       <property name="spacing" >
        <number>4</number>
       </property>
       <item row="0" column="0" >
        <widget class="qjackctlConnectView" native="1" name="MidiConnectView" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::TabFocus</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>4</number>
         </property>
         <property name="spacing" >
          <number>4</number>
         </property>
         <item>
          <widget class="QPushButton" name="MidiConnectPushButton" >
           <property name="toolTip" >
            <string>Connect currently selected ports</string>
           </property>
           <property name="text" >
            <string>&amp;Connect</string>
           </property>
           <property name="icon" >
            <iconset resource="qjackctl.qrc" >:/images/connect1.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="MidiDisconnectPushButton" >
           <property name="toolTip" >
            <string>Disconnect currently selected ports</string>
           </property>
           <property name="text" >
            <string>&amp;Disconnect</string>
           </property>
           <property name="icon" >
            <iconset resource="qjackctl.qrc" >:/images/disconnect1.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="MidiDisconnectAllPushButton" >
           <property name="toolTip" >
            <string>Disconnect all currently connected ports</string>
           </property>
           <property name="text" >
            <string>Disconnect &amp;All</string>
           </property>
           <property name="icon" >
            <iconset resource="qjackctl.qrc" >:/images/disconnectall1.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="MidiExpandAllPushButton" >
           <property name="toolTip" >
            <string>Expand all client ports</string>
           </property>
           <property name="text" >
            <string>E&amp;xpand All</string>
           </property>
           <property name="icon" >
            <iconset resource="qjackctl.qrc" >:/images/expandall1.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="MidiRefreshPushButton" >
           <property name="toolTip" >
            <string>Refresh current connections view</string>
           </property>
           <property name="text" >
            <string>&amp;Refresh</string>
           </property>
           <property name="icon" >
            <iconset resource="qjackctl.qrc" >:/images/refresh1.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="AlsaConnectTab" >
      <attribute name="title" >
       <string>MIDI-ALSA</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>4</number>
       </property>
       <property name="spacing" >
        <number>4</number>
       </property>
       <item row="0" column="0" >
        <widget class="qjackctlConnectView" native="1" name="AlsaConnectView" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::TabFocus</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>4</number>
         </property>
         <property name="spacing" >
          <number>4</number>
         </property>
         <item>
          <widget class="QPushButton" name="AlsaConnectPushButton" >
           <property name="toolTip" >
            <string>Connect currently selected ports</string>
           </property>
           <property name="text" >
            <string>&amp;Connect</string>
           </property>
           <property name="icon" >
            <iconset resource="qjackctl.qrc" >:/images/connect1.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="AlsaDisconnectPushButton" >
           <property name="toolTip" >
            <string>Disconnect currently selected ports</string>
           </property>
           <property name="text" >
            <string>&amp;Disconnect</string>
           </property>
           <property name="icon" >
            <iconset resource="qjackctl.qrc" >:/images/disconnect1.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="AlsaDisconnectAllPushButton" >
           <property name="toolTip" >
            <string>Disconnect all currently connected ports</string>
           </property>
           <property name="text" >
            <string>Disconnect &amp;All</string>
           </property>
           <property name="icon" >
            <iconset resource="qjackctl.qrc" >:/images/disconnectall1.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="AlsaExpandAllPushButton" >
           <property name="toolTip" >
            <string>Expand all client ports</string>
           </property>
           <property name="text" >
            <string>E&amp;xpand All</string>
           </property>
           <property name="icon" >
            <iconset resource="qjackctl.qrc" >:/images/expandall1.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="AlsaRefreshPushButton" >
           <property name="toolTip" >
            <string>Refresh current connections view</string>
           </property>
           <property name="text" >
            <string>&amp;Refresh</string>
           </property>
           <property name="icon" >
            <iconset resource="qjackctl.qrc" >:/images/refresh1.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4" />
 <customwidgets>
  <customwidget>
   <class>qjackctlConnectView</class>
   <extends>QWidget</extends>
   <header>qjackctlConnect.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ConnectionsTabWidget</tabstop>
  <tabstop>AudioConnectView</tabstop>
  <tabstop>AudioConnectPushButton</tabstop>
  <tabstop>AudioDisconnectPushButton</tabstop>
  <tabstop>AudioDisconnectAllPushButton</tabstop>
  <tabstop>AudioExpandAllPushButton</tabstop>
  <tabstop>AudioRefreshPushButton</tabstop>
  <tabstop>MidiConnectView</tabstop>
  <tabstop>MidiConnectPushButton</tabstop>
  <tabstop>MidiDisconnectPushButton</tabstop>
  <tabstop>MidiDisconnectAllPushButton</tabstop>
  <tabstop>MidiExpandAllPushButton</tabstop>
  <tabstop>MidiRefreshPushButton</tabstop>
  <tabstop>AlsaConnectView</tabstop>
  <tabstop>AlsaConnectPushButton</tabstop>
  <tabstop>AlsaDisconnectPushButton</tabstop>
  <tabstop>AlsaDisconnectAllPushButton</tabstop>
  <tabstop>AlsaExpandAllPushButton</tabstop>
  <tabstop>AlsaRefreshPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="qjackctl.qrc" />
 </resources>
 <connections/>
</ui>
